tests/load-tests/ci-scripts/utility_scripts/measure-time-skew.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Initialize sums
+sum_rtt=0
+sum_skew=0
+
+for i in {1..3}; do
+    # Step 1: Measure RTT in nanoseconds and calculate in seconds with fractional part
+    start=$(date +%s%N) # Start time in nanoseconds
+    oc exec prometheus-k8s-0 -n openshift-monitoring -- date -u +"%Y-%m-%dT%H:%M:%S%z" > /dev/null
+    end=$(date +%s%N)   # End time in nanoseconds
+
+    # Calculate round-trip time in seconds with fractions
+    rtt=$(echo "scale=6; ($end - $start) / 1000000000" | bc)
+    rtt=$(printf "%.3f" $rtt) # Round to three decimal places
+    sum_rtt=$(echo "$sum_rtt + $rtt" | bc)
+
+    # Step 2: Get remote time
+    remote_time=$(oc exec prometheus-k8s-0 -n openshift-monitoring -- date -u +"%Y-%m-%dT%H:%M:%S%z")
+    remote_time_epoch=$(date -ud "$remote_time" +"%s.%N") # Get seconds with nanoseconds
+
+    # Adjust for half of the round-trip time
+    latency_correction=$(echo "scale=6; $rtt / 2" | bc)
+    latency_correction=$(printf "%.3f" $latency_correction) # Round to three decimal places
+    remote_time_corrected_epoch=$(echo "scale=6; $remote_time_epoch - $latency_correction" | bc)
+    remote_time_corrected_epoch=$(printf "%.3f" $remote_time_corrected_epoch) # Round to three decimal places
+
+    # Step 3: Get local time
+    local_time=$(date -u +"%Y-%m-%dT%H:%M:%S%z")
```

Comment:
Should this call be right after `    remote_time=$(oc exec prometheus-k8s-0 -n openshift-monitoring -- date ...)` line?

Reply:
Hello. Here I meant that getting local time should happen as close to getting remote time as possible. Yes, in theory these commands between should be very fast, but still. We are measuring miliseconds here, so let's try to be precise.

Reply:
I understand
I'll call them one after the other